packages/workflow/src/components/style.scss

```diff
@@ -0,0 +1,229 @@
+@use "@wordpress/base-styles/mixins" as *;
+@use "@wordpress/base-styles/variables" as *;
+@use "@wordpress/base-styles/colors" as *;
+@use "sass:color";
+
+// dirty hack to clean up modal
```

Comment:
Ideally we would replace this comment with an explanation of what "clean up" is in this context.